### ISSUE TITLE
Quick and "Dirty" Line Width solution

### ISSUE THREAD

**Issue by SuberFu** [mrdoob/three.js#8716]
Not sure if this belong here.
This is my very initial attempt at contributing something.
This is trying to resolve the issue with line width maxing out at 1 on windows machine.
There are known solutions, namely to use polygons. However, I've some trouble finding a library that's well integrated into THREE.js.

So this is my first pass (hence, quick and "dirty") at creating one. The general idea is came from [here](https://mattdesl.svbtle.com/drawing-lines-is-hard).

[THREE.MeshLine](https://github.com/SuberFu/THREE.MeshLine/blob/df40c41f36d373e0334defaaa06d27c20ab6d925/MeshLine.js)
The THREE.MeshLine object can only take a normal geometry for now. During rendering, it converts that to a specialized buffergeometry that creates a quad at each line segment plus a quad at each joint (the quad at joint naturally creates a bevel line join).
THREE.MeshLine also can only take THREE.MeshLineMaterial, since it needs the special vertex shader (the three.js's shader system is both beautiful and too daunting for me to try to touch for now).
Setting linewidth on the material will change the width of line in the world coordinate space.
Setting linecolor on the material will change the line color if vertexColor is set to THREE.NoColors.

In short, the the user just need to create a new THREE.Geometry, fill in the vertices with the line, give it to THREE.MeshLine, and it will handle all the conversions automatically.

Obviously, there're a lot of improvements needed. Plus a few questions:
1. Is assigning the _bufferGeometry with a custom BufferGeometry a right way to go about this? Since the default BufferGeometry won't work with this instance.
2. Did I override the right things? It seems to be working, but I want to make sure I'm using the various methods as intended.

**Comment by jonnenauha** [comment]
Check out https://github.com/spite/THREE.MeshLine

**Comment by SuberFu** [comment]
Oh wow, now I feel stupid, didn't even thought about searching for that name...

**Comment by jonnenauha** [comment]
I was surprised your lib had the exact same name, first thought it was a fork :)

**Comment by SuberFu** [comment]
Just renamed it to MeshLineAlt, can't think of a better name.
